crates/turborepo-lib/src/task_hash.rs

```diff
@@ -164,7 +164,7 @@ pub struct TaskHashTrackerState {
     #[serde(skip)]
     package_task_outputs: HashMap<TaskId<'static>, Vec<AnchoredSystemPathBuf>>,
     #[serde(skip)]
-    package_task_cache: HashMap<TaskId<'static>, CacheResponse>,
+    package_task_cache: HashMap<TaskId<'static>, CacheResult<CacheHitMetadata>>,
```

Comment:
We can keep this as just `CacheHitMetadata` since we only ever insert cache hits
```suggestion
    package_task_cache: HashMap<TaskId<'static>, CacheHitMetadata>,
```